scripts/scripts.js

```diff
@@ -107,6 +162,7 @@ export function decorateMain(main) {
   decorateSections(main);
   decorateBlocks(main);
   decorateQuickLinks(main);
+  buildAppDowloadBlock(main);
```

Comment:
Should we be adding this in the? can't we keep the content directly in the doc with main section as we have kept. If not how is it different from the other blocks? 
<img width="1290" alt="image" src="https://github.com/aemsites/icicidirect/assets/38683124/bd68efb7-9594-44f8-8cfb-0d0cd3300c11">